packages/cli/src/qr/index.ts

```diff
@@ -0,0 +1,20 @@
+import bs58 from 'bs58'
+import { Buffer } from 'buffer'
+import qrcode from 'qrcode-terminal'
+
+export type ConnectionPayload = {
+  id: string
+  label: string
+  host: string
+  path: string
+}
+
+export const createConnectionQR = (connection: ConnectionPayload) => {
+  return new Promise((resolve) => {
+    const connectInfo = bs58.encode(Buffer.from(JSON.stringify(connection)))
```

Comment:
I think users may want to see the URL string, in addition to the QRCode. So we probably want a different API to support that